lib/model/message_list.dart

```diff
@@ -63,6 +64,22 @@ class MessageListMessageItem extends MessageListMessageBaseItem {
   });
 }
 
+/// An [OutboxMessage] to show in the message list.
+class MessageListOutboxMessageItem extends MessageListMessageBaseItem {
+  @override
+  final OutboxMessage message;
+  @override
+  final ZulipContent content;
+
+  MessageListOutboxMessageItem(
+    this.message, {
+    required super.showSender,
+    required super.isLastInBlock,
+  }) : content = ZulipContent(nodes: [
+    ParagraphNode(links: [], nodes: [TextNode(message.contentMarkdown)]),
```

Comment:
```suggestion
    ParagraphNode(links: null, nodes: [TextNode(message.contentMarkdown)]),
```

See the doc on the `links` field:
```
  /// A list of all [LinkNode] descendants.
  ///
  /// An empty list is represented as null.
  /// […]
```